<!DOCTYPE html>
<html lang='en'>
<head>
<title>osgQt_cmake.patch - smani/OpenSceneGraph-qt5-git/osgQt.git - smani/OpenSceneGraph-qt5-git/osgQt</title>
<meta name='generator' content='cgit v1.2.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='https://copr-dist-git.fedorainfracloud.org/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/atom/osgQt_cmake.patch?h=master' type='application/atom+xml'/>
<link rel='vcs-git' href='http://copr-dist-git.fedorainfracloud.org/git/smani/OpenSceneGraph-qt5-git/osgQt.git' title='smani/OpenSceneGraph-qt5-git/osgQt.git Git repository'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='smani/OpenSceneGraph-qt5-git/osgQt.git' href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/'>smani/OpenSceneGraph-qt5-git/osgQt.git</a></td><td class='form'><form method='get'>
<select name='h' onchange='this.form.submit();'>
<option value='f27'>f27</option>
<option value='f28'>f28</option>
<option value='f29'>f29</option>
<option value='f30'>f30</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' value='switch'/></form></td></tr>
<tr><td class='sub'>smani/OpenSceneGraph-qt5-git/osgQt</td><td class='sub right'></td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/'>summary</a><a href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/refs/'>refs</a><a href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/log/osgQt_cmake.patch'>log</a><a class='active' href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/tree/osgQt_cmake.patch'>tree</a><a href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/commit/osgQt_cmake.patch'>commit</a><a href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/diff/osgQt_cmake.patch'>diff</a><a href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/stats/osgQt_cmake.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/log/osgQt_cmake.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='search' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/tree/'>root</a>/<a href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/tree/osgQt_cmake.patch'>osgQt_cmake.patch</a></div><div class='content'>blob: 26051e1cfc3017469ffb8c9a039ecadcb63648b1 (<a href='/cgit/smani/OpenSceneGraph-qt5-git/osgQt.git/plain/osgQt_cmake.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
</pre></td>
<td class='lines'><pre><code>diff -rupN osgQt-6d324db8a56feb7d1976e9fb3f1de9bf7d255646/CMakeLists.txt osgQt-6d324db8a56feb7d1976e9fb3f1de9bf7d255646-new/CMakeLists.txt
<span class="hl kwa">--- osgQt-6d324db8a56feb7d1976e9fb3f1de9bf7d255646/CMakeLists.txt	2018-04-23 17:25:18.000000000 +0200</span>
<span class="hl kwb">+++ osgQt-6d324db8a56feb7d1976e9fb3f1de9bf7d255646-new/CMakeLists.txt	2018-05-16 20:42:33.898985126 +0200</span>
&#64;&#64; -872,7 +872,7 &#64;&#64; IF(BUILD_DOCUMENTATION)
         ${PROJECT_BINARY_DIR}/doc/OpenSceneGraphReferenceDocs/osg32-32.png COPYONLY
     )
     #INSTALL(FILES ${PROJECT_BINARY_DIR}/doc/${PROJECT_NAME}ReferenceDocs-${OPENSCENEGRAPH_VERSION}.chm DESTINATION doc OPTIONAL COMPONENT openscenegraph-doc)
<span class="hl kwa">-    INSTALL(DIRECTORY ${PROJECT_BINARY_DIR}/doc/OpenSceneGraphReferenceDocs DESTINATION doc COMPONENT openscenegraph-doc)</span>
<span class="hl kwb">+    #INSTALL(DIRECTORY ${PROJECT_BINARY_DIR}/doc/OpenSceneGraphReferenceDocs DESTINATION doc COMPONENT openscenegraph-doc)</span>
 
     # now set up openthreads documentation generation
     IF(BUILD_REF_DOCS_TAGFILE)
&#64;&#64; -884,7 +884,7 &#64;&#64; IF(BUILD_DOCUMENTATION)
         ${PROJECT_BINARY_DIR}/doc/OpenThreadsReferenceDocs/osg32-32.png COPYONLY
     )
     #INSTALL(FILES ${PROJECT_BINARY_DIR}/doc/${PROJECT_NAME}ReferenceDocs-${OPENSCENEGRAPH_VERSION}.chm DESTINATION doc OPTIONAL COMPONENT openscenegraph-doc)
<span class="hl kwa">-    INSTALL(DIRECTORY ${PROJECT_BINARY_DIR}/doc/OpenThreadsReferenceDocs DESTINATION doc COMPONENT openthreads-doc)</span>
<span class="hl kwb">+    #INSTALL(DIRECTORY ${PROJECT_BINARY_DIR}/doc/OpenThreadsReferenceDocs DESTINATION doc COMPONENT openthreads-doc)</span>
 
     # Process our other doxyfiles but don&#39;t create targets for these
     CONFIGURE_FILE(${PROJECT_SOURCE_DIR}/doc/Doxyfiles/all_Doxyfile
diff -rupN osgQt-6d324db8a56feb7d1976e9fb3f1de9bf7d255646/CMakeModules/OsgMacroUtils.cmake osgQt-6d324db8a56feb7d1976e9fb3f1de9bf7d255646-new/CMakeModules/OsgMacroUtils.cmake
<span class="hl kwa">--- osgQt-6d324db8a56feb7d1976e9fb3f1de9bf7d255646/CMakeModules/OsgMacroUtils.cmake	2018-04-23 17:25:18.000000000 +0200</span>
<span class="hl kwb">+++ osgQt-6d324db8a56feb7d1976e9fb3f1de9bf7d255646-new/CMakeModules/OsgMacroUtils.cmake	2018-05-16 20:42:33.898985126 +0200</span>
&#64;&#64; -463,7 +463,7 &#64;&#64; MACRO(SETUP_EXAMPLE EXAMPLE_NAME)
         IF(APPLE)
             INSTALL(TARGETS ${TARGET_TARGETNAME} RUNTIME DESTINATION share/OpenSceneGraph/bin BUNDLE DESTINATION share/OpenSceneGraph/bin )
         ELSE(APPLE)
<span class="hl kwa">-            INSTALL(TARGETS ${TARGET_TARGETNAME} RUNTIME DESTINATION share/OpenSceneGraph/bin COMPONENT openscenegraph-examples )</span>
<span class="hl kwb">+            INSTALL(TARGETS ${TARGET_TARGETNAME} RUNTIME DESTINATION bin COMPONENT openscenegraph-examples )</span>
             IF(MSVC AND NOT CMAKE_BUILD_TYPE STREQUAL &quot;Release&quot;)
                 INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET_TARGETNAME}${CMAKE_BUILD_POSTFIX}.pdb DESTINATION share/OpenSceneGraph/bin COMPONENT openscenegraph-examples)
             ENDIF(MSVC AND NOT CMAKE_BUILD_TYPE STREQUAL &quot;Release&quot;)
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated by <a href='https://git.zx2c4.com/cgit/about/'>cgit v1.2.1</a> (<a href='https://git-scm.com/'>git 2.18.0</a>) at 2020-04-18 16:11:18 +0000</div>
</div> <!-- id=cgit -->
</body>
</html>
